cypress.config.js

```diff
@@ -2,8 +2,8 @@ const { defineConfig } = require('cypress');
 
 module.exports = defineConfig({
   e2e: {
+    baseUrl: 'https://demoqa.com/alerts',
     setupNodeEvents(on, config) {
-      // implement node event listeners here
     }
```

Comment:
The `setupNodeEvents` function is currently empty. If you don't plan to use it, it's better to remove it to avoid confusion. If you do plan to use it later, consider adding a comment about its future use.